core/src/main/java/com/linecorp/armeria/server/docs/DocService.java

```diff
@@ -164,7 +164,7 @@ public void serverStarting(Server server) throws Exception {
                 spec = addDocStrings(spec, services);
                 spec = addExamples(spec);
 
-                final List<Version> versions = ImmutableList.copyOf(
+                final List<Version> versions = ImmutableList.sortedCopyOf(
```

Comment:
How about just modifying `Version.identify()` to use a `SortedMap` rather than a `HashMap`?

Reply:
This seems like the simplest solution!